README.md

```diff
@@ -140,3 +140,29 @@ CoreDNS is a fast and flexible DNS server. It has a focus of service discovery i
 -	Recommended Skills: Golang, Microsoft Azure
 -	Mentor(s): Yong Tang (@yongtang)
 
+### TiKV
+
+TiKV is an open-source distributed transactional Key-Value database. [https://tikv.org](https://tikv.org).
+
+#### Optimize Raft in cross datacenter scenarios
+
+- Description: The original Raft consensus algorithm is not optimized for cross data center scenarios, where we cannot avoid high latency between two data centers but alleviate it. The goal for this section is to introduce some optimizations from Paxos that help us enhance Raft.
+- Recommended Skills: Rust, Raft
+- Mentor(s): Ana Hobden (@hoverbear), JianJun Li (@busyjay), Nick Cameron (@nrc)
+- Issue: https://github.com/pingcap/raft-rs/issues/178, https://github.com/pingcap/raft-rs/issues/145, https://github.com/pingcap/raft-rs/issues/136, https://github.com/pingcap/raft-rs/issues/135
```

Comment:
@siddontang I could be wrong here, but this reads to me as a project idea for raft-rs for pingcap and not the upstream tikv project. Can you clarify how this would include changes to the upstream project?

(Asking this because GSoC is only allowed for upstream projects)

Reply:
@nikhita 

TiKV heavily depends on Raft consensus algorithm (pingcap/raft-rs), and optimizing Raft can improve TiKV directly.

If you are confused with this, I will create the issues in TiKV repository directly.